test/Microsoft.IdentityModel.Logging.Tests/project.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
     "version": "1.0.0-*",
     "dependencies": {
-        "System.IdentityModel.Tokens.Jwt": "5.0.0-*",
+        "System.IdentityModel.Tokens.Tests": "5.0.0-*",
```

Comment:
This is to use IdentityUtilities and ClaimSets classes in TextWriterEventListenerLogging method.